.travis.yml

```diff
@@ -56,28 +60,24 @@ script:
   - make update-bazel
   - make verify
   - make test-ci
-  - |
-    kind create cluster --image="kindest/node:${KUBE_VERSION}" --name="$KIND_CLUSTER_NAME"
-    export KUBECONFIG="$(kind get kubeconfig-path --name="$KIND_CLUSTER_NAME")"
-    export KUBECONTEXT="kubernetes-admin@$KIND_CLUSTER_NAME"
-  - |
-    kubectl create clusterrolebinding tiller-cluster-admin \
+  - kind create cluster --image="kindest/node:${KUBE_VERSION}" --name="$KIND_CLUSTER_NAME"
+  - export KUBECONFIG="$(kind get kubeconfig-path --name="$KIND_CLUSTER_NAME")"
+  - export KUBECONTEXT="kubernetes-admin@$KIND_CLUSTER_NAME"
+  - kubectl create clusterrolebinding tiller-cluster-admin \
       --clusterrole=cluster-admin \
       --serviceaccount=kube-system:default
-    helm init
+  - helm init
   # run integration tests while tiller is starting
   - make integration-test-ci
-  - |
-    helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
-    helm install svc-cat/catalog --name catalog --namespace catalog
-    helm install svc-cat/ups-broker --name ups-broker --namespace ups-broker
+  - helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+  - helm install svc-cat/catalog --name catalog --namespace catalog
+  - helm install svc-cat/ups-broker --name ups-broker --namespace ups-broker
   # build container and run checks while service catalog and UPS are starting
   - make docker
   - make check
-  - |
-    kubectl create -f it/sc/ups-clusterservicebroker.yaml
-    sleep 5
-    make integration-test-sc-ci
+  - kubectl create -f it/sc/ups-clusterservicebroker.yaml
+  - sleep 5
```

Comment:
there should be more reliable way of checking that broker has been processed - check it's status?